ledger/participant-integration-api/src/main/scala/platform/store/backend/common/EventStorageBackendTemplate.scala

```diff
@@ -121,7 +125,8 @@ abstract class EventStorageBackendTemplate(
       array[Int]("create_observers") ~
       str("create_agreement_text").? ~
       byteArray("create_key_value").? ~
-      int("create_key_value_compression").?
+      int("create_key_value_compression").? ~
+      hashFromHexString("create_key_hash").?
```

Comment:
Just for my own curiousity: Why do we store the key hash as a hex string rather than as a binary blob? I would have assumed that the latter is preferable  wrt to storage in the db.

Reply:
I was wondering the same. The hex string uses twice as much space, and we have to encode/decode it to/from a hex string. Probably just an oversight.

Reply:
FYI Oracle cannot have btrees over blob. This column index is driving the key lookups. Same (evil?) forces at action with the hex offsets - with similar impediments.

Reply:
Thanks for the explanation, @nmarton-da !